app/controllers/api/base_controller.rb

```diff
@@ -59,14 +55,6 @@ def response_with_mfa_warning(response)
     message
   end
 
-  def mfa_recommended_not_yet_enabled?
-    user_mfa_recommended? && @api_key.user&.mfa_disabled?
-  end
-
-  def mfa_recommended_weak_level_enabled?
-    user_mfa_recommended? && @api_key.user&.mfa_ui_only?
-  end
```

Comment:
I find it difficult to review PRs where we’ve changed our mind about the implementation partway through (or, at least, I _think_ that’s what’s happened — I’m not fully confident). Can we revise this PR to contain commits which all work in a single clear direction?